src/link/stm32_flash_h750_exst.ld

```diff
@@ -35,7 +35,8 @@ ENTRY(Reset_Handler)
 
 For H7 EXST (External Storage) targets a binary is built that is placed on an external device.
 The bootloader will then copy this entire binary to RAM, at the CODE_RAM address.  The bootloader
-then executes code at the CODE_RAM address.
+then executes code at the CODE_RAM address. The address of CODE_RAM is fixed to 0x24010000
```

Comment:
Thanks for adding this - I think it's best to also add a comment to the actual definition, to make sure this is not moved again accidentally.